app/models/notification_recipient.rb

```diff
@@ -17,6 +17,7 @@ def payload
       :subject    => notification_blueprint.subject,
       :created_at => notification.created_at,
       :group      => notification_blueprint.group,
+      :actions    => notification_blueprint.actions
```

Comment:
Actions have nothing to do with subject - it's just a dumb set of links for the moment, it can be any hash https://github.com/theforeman/foreman/pull/4265/files#diff-61072566998937ff274a277f94d284e8R12

It can be made more complicated but I'm not sure how good is it to start coding logic nothing is using, this approach works well with the RSS PR (if passing an array of links isn't good enough ,e.g: you want to pass the HTTP action - POST, etc.., that's easily changeable if we need to do it)